scripts/etc/init.d/rcS

```diff
@@ -85,6 +85,25 @@ sanitize_inputfile()
     fi
 }
 
+install_files()
+{
+  file_to_read=$1
+  sanitize_inputfile file_to_read
```

Comment:
Well, there's no danger here, we're not using arrays and the rest of the file doesn't use curly braces either. I want to keep it like this.
If you want to add curly braces we should do all scripts and that is/should be IMO a separate issue.